### PR TITLE
Pikabu changes

### PR DESCRIPTION
**Pull request by kpeatt**
Status: **Opened for visibility**
Reviewers: @scalvert @tedtate 
## Changes
- Add an `append` function so that shade is not strictly tied to existing at root level (and so works with Pikabu)
- Made Shades animation run outside of the queue so that it does not delay other animations or vice versa.
## Todo
- [ ] Add test for `append` function
- [ ] Update Pinny README with new info
- [ ] Update changelog entry and bump version

### CODE REVIEW

**Comment by kpeatt** [comment]
We should probably add a test for the append function as I didn't do that.

File: src/js/shade.js [hunk 1]
```diff
@@ -92,6 +93,7 @@
                     opacity: this.options.opacity
                 },
                 {
+                    queue: false,
```
Comment (scalvert) [inline src/js/shade.js:97]: Was this required for pikabu?

Comment (kpeatt) [inline src/js/shade.js:97]: It's actually a good change in general — it fixes the animation jank when shade was animating at a different duration than pinny/pikabu. It's necessary for pikabu though.

File: src/js/shade.js [hunk 2]
```diff
@@ -24,7 +24,7 @@
         Shade.__super__.call(this, element, options, Shade.DEFAULTS);
     }
 
-    Shade.VERSION = '0';
+    Shade.VERSION = '1.1.1';
```
Comment (tedtate) [inline src/js/shade.js]: Pretty sure this line shouldn't be there in the first place, it should be added into our dist file.

Comment (kpeatt) [inline src/js/shade.js]: Mm yep this was a c/p error.

**Comment by tedtate** [comment]
Updated OP with what we need to get finished to merge this PR.

**Comment by donnielrt** [comment]
Hey, we’re looking to prune older unattended PRs. If this PR is still relevant and you would like to see it merged in, please reopen the PR and we’ll add it to our backlog! Thanks!

**Pull request closed without merging**